gravityflow/gravityflow
# Add support for reports in the shortcode

This PR adds support for reports in the `[gravityflow]` shortcode. When set `page="reports"`, there are four newly added attributes can be set with it: `range`, `category`, `step_id` and `assignee`. Users can also specify a default form with the attribute `form`.

## Testing instructions
1. Create a new page and insert the shortcode `[gravityflow page="reports"]`.
2. Test if `range`, `category`, `step_id`, `assignee` and `form` attributes can work as expected.
3. Test if the filters can work as expected.
